doc/nrf/libraries/networking/nrf_cloud.rst

```diff
@@ -140,7 +140,7 @@ For example, a device that supports all the FOTA types writes the following data
                ]
    }}}}}
 
-You can initiate FOTA updates through `nRF Connect for Cloud`_ or by using the `nRF Connect for Cloud Device API`_.
+You can initiate FOTA updates through `nRF Cloud`_ or by using the `nRF Cloud Device API`_.
```

Comment:
We are deprecating the use of the name "nRF Cloud Device API" in favor of "nRF Cloud REST API."